public/spec/features/cite_spec.rb

```diff
@@ -27,7 +27,7 @@
     cite_item_btn.click
     item_input = page.find('#tempItemInput')
     item_input.click
-    if page.driver.browser.capabilities.platform == 'mac'
+    if page.driver.browser.capabilities.platform_name =~ /^mac/
```

Comment:
I do not have access to a mac, could someone run this spec on a mac locally?

Reply:
Works on my Mac